importer/Monument.py

```diff
@@ -261,7 +264,7 @@ def create_wlm_source(self, monuments_all_id):
     def exists_with_monument_article(self,
                                      language,
                                      article_keyword="monument_article"):
-        """Check whether the data object has anythin in the monument_article field."""
+        """Get the Q-item connected to monument_article (or equivalent), if any."""
```

Comment:
Use either "Wikidata item" or "Qid/Q-id of item" since Q-item isn't a term used elsewhere.